stdlib/public/runtime/LibPrespecialized.cpp

```diff
@@ -108,3 +113,50 @@ swift::getLibPrespecializedMetadata(const TypeContextDescriptor *description,
             result, (int)key.size(), key.data());
   return result;
 }
+
+void _swift_validatePrespecializedMetadata(unsigned *outValidated,
+                                           unsigned *outFailed) {
```

Comment:
Extremely minor nit: This being C++, you could have returned an `std::tuple<>` or `std::pair<>`.

Reply:
I wanted a C interface so I could more easily call it from random places, like the debugger, or the test.

Reply:
Ah, makes sense.

Reply:
It occurs to me that the use of out parameters is going to make this kind of inconvenient to call from the debugger. Maybe I can improve that.

Reply:
Maybe return a two-element struct instead?